department-of-veterans-affairs/caseflow
# Div/appeals 35484

Resolves both [Correspondence Cases - Unassigned Tab: Batch Assignment](https://jira.devops.va.gov/browse/APPEALS-35484)
[Correspondence Cases - Assigned Tab: Batch Assignment](https://jira.devops.va.gov/browse/APPEALS-36209)

# Description
Provided the ability to manually assign correspondences that are under the assigned and unassigned tab for this PR.

## Acceptance Criteria

## Testing Plan
1. Login with [MAIL_TEAM_SUPERVISOR_MAIL_INTAKE_USER (VACO)]
2. Click [Your Queue] page
3. From the switch views dropdown, navigate to correspondence cases.
4. From there, navigate to Unassigned Tab
5. You will see the drop-down box along with the assign button that is in the disabled state.
6. Select the checkboxes and the mail team user from the drop-down and then click Assign
7. Also navigate to Assigned Tab
8. You will see the drop-down box along with the Reassign button that is in the disabled state.
9. Select the checkboxes and the mail team user from the drop-down and then click Reassign.


# Frontend
## User Facing Changes
 
<img width="1639" alt="Screenshot 2024-02-29 at 2 44 58 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/135847343/5546c7f9-c29b-4262-9cd7-25a2b737ff94">
<img width="1084" alt="Screenshot 2024-02-29 at 2 45 19 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/135847343/1f3a6b42-1155-4470-aa9a-d81871a875d2">
<img width="1548" alt="Screenshot 2024-02-29 at 2 45 37 PM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/135847343/5164e3a0-ec20-4e50-9202-55001389bca6">
[
<img width="1523" alt="Screenshot 2024-03-01 at 9 21 46 AM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/135847343/ec6a4a40-ad8c-4609-aa10-70efd0d96465">
<img width="1153" alt="Screenshot 2024-03-01 at 9 22 05 AM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/135847343/10f47684-4d2b-4db5-96d5-3f08a7ee313f">
](url)

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
